src/Common/Commands.Common/Commands.Common.csproj

```diff
@@ -103,12 +103,16 @@
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\..\packages\Microsoft.WindowsAzure.Management.4.1.1\lib\net40\Microsoft.WindowsAzure.Management.dll</HintPath>
     </Reference>
+    <Reference Include="Microsoft.WindowsAzure.Storage">
```

Comment:
I am not sure why we need to regenerate the wix file. We haven't added any new project/assembly. Just re-using an existing one(Microsoft.WindowsAzure.Management) and added it as a reference to the Command.Common project.